sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -601,6 +578,27 @@ public OffsetDateTime renewSessionLock(String sessionId) {
         return asyncClient.renewSessionLock(sessionId).block(operationTimeout);
     }
 
+    /**
+     * Starts the auto lock renewal for a session with the given lock.
```

Comment:
`with the given lock` : It is a bit confusing since user never gave us any lock in this API. 
Suggestion `Starts the auto lock renewal for a session id.`